### ISSUE TITLE
Allow searches for non-equality

### ISSUE THREAD

**Issue by IanWhitney** [umn-asr/query_string_search#17]
Example query strings:

```
movies?q=year!=1994
```

All movies whose year is not 1994.

```
movies?q=year!=1994|1993
```

All movies whose year is not 1994 or 1993.

```
movies?q=home_formats!=BD
```

All movies that have not been released on Blue Ray.

```
movies?q=home_formats!=BD|Amazon
```

All movies that have not been released on Blue Ray or Amazon.

**Comment by IanWhitney** [comment]
@cdinger

Once this feature is implemented and the Course Data Service is using the updated gem, then users of that service will be able to search for courses that do not have a WI course attribute.

**Comment by IanWhitney** [comment]
Implemented in #18, deployed in 0.0.7
